.eslintrc.json

```diff
@@ -6,5 +6,8 @@
   "root": true,
   "settings": {
     "import/resolver": "webpack"
+  },
+  "rules": {
+    "import/extensions": "off"
```

Comment:
I'd prefer not to turn this rule off entirely.  What about something like `["warn", "never", { "scss": "always" }]` instead?